### PR TITLE
LLM: adjust portable zip content

### PR DESCRIPTION
**Pull request by JinBridger**
## Description

Adjust the portable zip content. Rename `portable-executable` to `portable-zip` and adjust the content of `README.md`

### CODE REVIEW

File: python/llm/portable-zip/README.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# BigDL-LLM Portable Zip For Windows: User Guide
+
+## Introduction
+
+This portable zip includes everything you need to run LLM (except models). Please refer to How to use section to get started.
```
Comment (Oscilloscope98) [inline python/llm/portable-zip/README.md:5]: We could add a jump link here for How to use

sth like
```
[How to use](#how-to-use)
```

File: python/llm/portable-zip/README.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+## How to use
+
+1. Download the zip from link [here]().
+2. (Optional) You could also build the zip on your own. Run `setup.bat` and it will generate the zip file.
+3. Unzip `bigdl-llm.zip`.
+4. Download the model to your computer. Please ensure there is a file named `config.json` in the model folder, otherwise the script won't work.
+
+   ![](https://llm-assets.readthedocs.io/en/latest/_images/one-click-installer-user-guide-step1.png)
+
+5. Run `chat.bat` in Terminal and input the path of the model (e.g. `path\to\model`, note that there's no slash at the end of the path). Press Enter and wait until model finishes loading. Then enjoy chatting with the model!
```
Comment (Oscilloscope98) [inline python/llm/portable-zip/README.md:31]: Add description that we need to go into the unzipped folder, and then run `chat.bat`

Or I think we could even double click to open the `bat.py` file directly?

File: python/llm/portable-zip/README.md (hunk 1 above)
Comment (Oscilloscope98) [reply python/llm/portable-zip/README.md:5]: `This portable zip includes everything you need to run an LLM with BigDL-LLM optimizations (except models) . Please refer to [How to use](#how-to-use) section to get started.`

Comment (JinBridger) [reply python/llm/portable-zip/README.md]: Already done as you said : )

File: python/llm/portable-zip/README.md (hunk 2 above)
Comment (JinBridger) [reply python/llm/portable-zip/README.md]: Sure. I add `pause` at the end of bat and you could double click it to run.

**Review by Oscilloscope98 — APPROVED** [review]
LGTM. Merged for now. Will raise another PR for any further modification.

**Pull request merged**